ingestion/src/metadata/orm_profiler/orm/functions/median.py

```diff
@@ -47,10 +47,11 @@ def _(elements, compiler, **kwargs):
     return "median(%s)" % col
 
 
+@compiles(MedianFn, Dialects.Trino)
 @compiles(MedianFn, Dialects.Presto)
```

Comment:
yeah for the future, Trino & Presto usually go hand in hand. Did not notice on the last PR